lib/modules/manager/argocd/extract.ts

```diff
@@ -66,28 +99,22 @@ function createDependency(
   };
 }
 
-export function extractPackageFile(
-  content: string,
-  fileName: string,
-  _config?: ExtractConfig
-): PackageFileContent | null {
-  // check for argo reference. API version for the kind attribute is used
-  if (fileTestRegex.test(content) === false) {
-    return null;
+export function processAppSpec(
+  spec: ApplicationSpec | null | undefined
+): PackageDependency[] {
+  if (is.nullOrUndefined(spec)) {
+    return [];
   }
 
-  let definitions: ApplicationDefinition[];
-  try {
-    definitions = loadAll(content) as ApplicationDefinition[];
-  } catch (err) {
-    logger.debug({ err, fileName }, 'Failed to parse ArgoCD definition.');
-    return null;
+  const deps: Array<PackageDependency | null> = [];
```

Comment:
please, we told multiple times to not use that array syntax!
```suggestion
  const deps: (PackageDependency | null)[] = [];
```